core/store/src/db.rs

```diff
@@ -363,8 +367,22 @@ fn rocksdb_options() -> Options {
     opts.set_bytes_per_sync(1048576);
     opts.set_write_buffer_size(1024 * 1024 * 512 / 2);
     opts.set_max_bytes_for_level_base(1024 * 1024 * 512 / 2);
-    opts.increase_parallelism(cmp::max(1, num_cpus::get() as i32 / 2));
-    opts.set_max_total_wal_size(1 * 1024 * 1024 * 1024);
+    #[cfg(not(feature = "single_thread_rocksdb"))]
+    {
+        println!("Use background threads in rocksdb");
```

Comment:
Should we remove this log message or convert it into a proper tracing::info/debug/trace?

Reply:
Will convert to debug

Reply:
Seems we didn't log anything in db config module in neard, so I'll remove it to be consistent